controllers/sriovnetworknodepolicy_controller.go

```diff
@@ -553,6 +554,13 @@ func (r *SriovNetworkNodePolicyReconciler) renderDevicePluginConfigData(pl *srio
 			if len(p.Spec.NicSelector.PfNames) > 0 {
 				netDeviceSelectors.PfNames = sriovnetworkv1.UniqueAppend(netDeviceSelectors.PfNames, p.Spec.NicSelector.PfNames...)
 			}
+			if p.Spec.LinkType != "" && !sriovnetworkv1.StringInArray(p.Spec.LinkType, netDeviceSelectors.LinkTypes) {
```

Comment:
!sriovnetworkv1.StringInArray(p.Spec.LinkType, netDeviceSelectors.LinkTypes) this is to check if that linktype selector already exist?